nemli/config.py

```diff
@@ -0,0 +1,11 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    model_config = SettingsConfigDict(env_prefix="NEMLI_")
```

Comment:
```suggestion
    model_config = SettingsConfigDict(env_prefix="NEMLI_", env_nested_delimiter="__")
```

`Discord` é um `BaseModel` separado que dentro dele tem `token` e `prefix`

_"Deixa que o namespace builda a separação pra vc, nengue"_

Reply:
Pseudocódigo aqui rapidinho

```py

class DiscordSettings(BaseModel):
    token: str
    prefix: str


class NemliSettings(BaseSettings):
    model_config = SettingsConfigDict(env_prefix="NEMLI_", nested bla bla bla)

    discord: DiscordSettings
```

Reply:
Corrigido em: https://github.com/milharal-dev/nemli-nemlerei-bot/pull/1/commits/44cc82e233311cbb4033e0a5717fde8275a025af @flipbit03

Reply:
corrigido em: https://github.com/milharal-dev/nemli-nemlerei-bot/pull/1/commits/44cc82e233311cbb4033e0a5717fde8275a025af @flipbit03